models/db.go

```diff
@@ -10,6 +10,7 @@ type Datastore interface {
 	GetNotebook(notebookTitle string) (Notebook, error)
 	GetAllNotebooks() ([]Notebook, error)
 	AddNote(notebookTitle string, note Note) error
+	DeleteNote(notebookName string, noteId uint64) error
```

Comment:
I kept the parameter convention the same, but why even pass in a `uint64` if the argument from the command line is a string and we end up converting it to a `[]byte` anyway?

Reply:
Good call. The id can just be a `[]byte` instead.